apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -605,7 +617,8 @@ export class ChangeAction extends BaseRushAction {
   }
 
   private async _writeChangeFile(
-    promptModule: inquirer.PromptModule,
+    // eslint-disable-next-line @typescript-eslint/typedef
```

Comment:
As I explained with `IYarnLockfileParseResult`, just use a regular `import` with a TODO comment saying to convert to  "import type" later.  *Don't remove our type annotations.*